bin/kibana.bat

```diff
@@ -22,7 +22,7 @@ If Not Exist "%NODE%" (
 )
 
 TITLE Kibana Server
-"%NODE%" %NODE_OPTIONS% "%DIR%\src\cli" %*
+"%NODE%" --max-old-space-size=256 %NODE_OPTIONS% "%DIR%\src\cli" %*
```

Comment:
I need to update this batch file to reflect the changes made in `bin/kibana`. Downloading a Windows VM to verify before adding to the PR.